guides/common/modules/proc_deploying-the-ca-certificate-on-a-host-by-using-ansible-rex.adoc

```diff
@@ -0,0 +1,33 @@
+[id="deploying-the-ca-certificate-on-a-host-by-using-ansible-rex"]
+= Deploying the CA certificate on a host by using Ansible REX
```

Comment:
Do we use the term "Ansible REX" elsewhere? I'd lean to keeping it simple use `using Remote Exection` instead (as we do elsewhere).

Reply:
I needed to distinguish the Ansible/Script provider and the title becomes too long if I spell it out.